dockerfiles/hpcc/templates/thor.yaml

```diff
@@ -5,15 +5,15 @@
 {{- $thorAgentName := printf "%s-thoragent" .name }}
 {{- $slaveName := printf "%s-slave" .name }}
 {{- $serviceName := printf "%s-svc" .name }}
-{{- $eclAgentDefaults := dict "name" $eclAgentName "replicas" 1 }}
+{{- $eclAgentDefaults := dict "name" $eclAgentName "useChildProcesses" true "replicas" 1 }}
```

Comment:
if we are defaulting off in general, why not here too?

Reply:
Because the "workflow" agent running thor jobs is assumed to be lightweight